code/datums/diseases/advance/presets.dm

```diff
@@ -39,4 +39,21 @@
 			symptoms += S
 	Refresh()
 
-	name = "Sample #[rand(1,10000)]"
\ No newline at end of file
+	name = "Sample #[rand(1,10000)]"
+
+/datum/disease/advance/pharaoh
+	copy_type = /datum/disease/advance
+
+/datum/disease/advance/pharaoh/New()
+	name = "Curse of the Pharaoh"
+
+	symptoms = list(new/datum/symptom/fever, new/datum/symptom/sneeze, new/datum/symptom/viralevolution)

```

Comment:
Could you un-neuter sneezing to make the disease more effective?

Is there a way to somehow make a viral bonding autosurgeon?